notebooks/blog_materials/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/1_preprocess.ipynb

```diff
@@ -62,7 +62,8 @@
    "source": [
     "url = 'https://archive.ics.uci.edu/ml/machine-learning-databases/00432/Data/News_Final.csv'\n",
     "r = requests.get(url, allow_redirects=True, verify=False)\n",
-    "open('data/News_Final.csv', 'wb').write(r.content)"
+    "open('data/News_Final.csv', 'wb').write(r.content)\n",
+    "r.close()"
```

Comment:
I meant the opened local file ("News_Final.csv") has to be closed after written, rather than closing the requests.response object.
r.close() may not be needed.

sample:

with open('data/News_Final.csv', 'wb') as fd:
    fd.write(r.content)